webpack.config.js

```diff
@@ -1,22 +1,35 @@
+/* eslint-disable */
+var ExtractTextPlugin = require("extract-text-webpack-plugin");
+
 module.exports = {
   entry: './src/frontend/index.jsx',
   devtool: 'source-map',
   output: {
-    path: './src/front/js/',
-    filename: 'build.js'
+    path: './dist/',
+    filename: 'bundle.js'
   },
   resolve: {
-      modulesDirectories: ["node_modules"],
-      extensions: ["", ".js", ".jsx"]
+    modulesDirectories: ["node_modules"],
+    extensions: ["", ".js", ".jsx"]
   },
   module: {
-      loaders: [{
-          test: /\.jsx|\.js$/,
-          exclude: /node_modules/,
-          loader: "babel"
-      }, {
-          test: /\.json$/,
-          loader: "json"
-      }]
+    loaders: [
+      {
+        test: /\.jsx|\.js$/,
+        exclude: /node_modules/,
+        loader: "babel"
+      },
+      {
+        test: /\.json$/,
+        loader: "json"
+      },
+      {
+        test: /\.scss$/,
+        loader: ExtractTextPlugin.extract("style", "css?sourceMap!sass?sourceMap"),
```

Comment:
One tip for the future - if you use preprocessor features like include or mixins while importing separate, component specific style files - nothing, even ExtractTextPlugin will save you from duplication of mixin code.